Sources/Private/CoreAnimation/Animations/StrokeAnimation.swift

```diff
@@ -59,6 +59,9 @@ extension CAShapeLayer {
         try KeyframeGroup(keyframes: $0)
           .exactlyOneKeyframe(context: context, description: "stroke dashPattern").value.cgFloatValue as NSNumber
       }
+      if lineDashPattern?.allSatisfy({ $0.floatValue.isZero }) ?? false {
```

Comment:
I personally find `== true` more idiomatic than `?? false` (subjective for sure though!)

```suggestion
      if linePattern.allSatisfy({ $0.floatValue.isZero }) == true {
```

Reply:
exactily, I changed 👍